user_front/components/footer.vue

```diff
@@ -0,0 +1,19 @@
+<template>
+  <div class="footer">
+    ︎©︎長岡技術科学大学技大祭実行員
+  </div>
+</template>
+
+<style>
+  .footer{
+    font-family: Noto Sans JP;
+    @apply 
+    flex justify-center items-center 
+    absolute inset-x-0 bottom-0
+    border-t-2 mx-4
+    text-xs
+    h-[35px]
+    font-light
+    leading-4 ;
+  }
+</style>
```

Comment:
```suggestion
</style>

```
ファイルの一番下は空行を追加しましょう
vscodeで自動でやってくれるので設定してください。